src/utils/getFilterdTodos.tsx

```diff
@@ -0,0 +1,26 @@
+import { SelectedType } from '../types/SelectedType';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  selectedOption: SelectedType,
+) => {
+  let filteredTodos = [...todos];
+
+  if (selectedOption !== SelectedType.ALL) {
+    filteredTodos = filteredTodos.filter(todo => {
```

Comment:
```suggestion
  if (selectedOption !== SelectedType.ALL) {
    todos.filter(todo => {
```
- `filter` doesn't mutate an array, so you don't need a copy
- You don't need to assign to a variable, you can just return values from function